### PR TITLE
Containerise MWS panel

### PR DESCRIPTION
**Pull request by msb**

### CODE REVIEW

File: docker/ssh_config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Host *
```
Comment (rjw57) [inline docker/ssh_config]: Why is this necessary?

File: mws/apimws/utils.py [hunk 2]
```diff
@@ -137,3 +138,9 @@ def domain_confirmation_user(domain_name):
         to=[domain_name.vhost.service.site.email],
         headers={'Return-Path': getattr(settings, 'EMAIL_MWS3_SUPPORT', 'mws-support@uis.cam.ac.uk')}
     ).send()
+
+
+def execute_userv_process(userv_cmd, **kwargs):
+    """Executes a userv command on the host via ssh"""
```
Comment (rjw57) [inline mws/apimws/utils.py]: s/host/panel/? "host" suggests the xen cluster to me.

File: docker/ssh_config (hunk 1 above)
Comment (msb) [reply docker/ssh_config]: This is to avoid the "The authenticity of host '????' can't be established" question when using ssh via python.

Comment (rjw57) [reply docker/ssh_config]: Could we perhaps poke them into ~/.ssh/known_hosts then? Rather than just trusting our network?

**Comment by abrahammartin** [comment]
Before I review it, it's probably worth resolving the conflicts first, just in case resolving the conflicts modifies thing significantly. @msb

**Comment by ph448** [comment]
closing as this has been stale and will probably need re-doing from scratch
